parsl/monitoring/db_manager.py

```diff
@@ -279,20 +279,20 @@ class Resource(Base):
 class DatabaseManager:
     def __init__(self,
                  db_url: str = 'sqlite:///runinfo/monitoring.db',
-                 logdir: str = '.',
+                 run_dir: str = '.',
                  logging_level: int = logging.INFO,
                  batching_interval: float = 1,
                  batching_threshold: float = 99999,
                  ):
 
         self.workflow_end = False
         self.workflow_start_message: Optional[MonitoringMessage] = None
-        self.logdir = logdir
-        os.makedirs(self.logdir, exist_ok=True)
+        self.run_dir = run_dir
+        os.makedirs(self.run_dir, exist_ok=True)
 
         logger.propagate = False
 
-        set_file_logger("{}/database_manager.log".format(self.logdir), level=logging_level,
+        set_file_logger("{}/database_manager.log".format(self.run_dir), level=logging_level,
```

Comment:
In at least one other spot in this PR, this style was transformed to f-string syntax.  A consideration for here as well?  Or vice-versa, and undo the other hunk?